app/views/layouts/application.html.erb

```diff
@@ -28,10 +28,15 @@
     <% if @meta_section %>
       <meta name="govuk:section" content="<%= @meta_section %>">
     <% end %>
+    <%= cookieless_variant.analytics_meta_tag.html_safe %>
```

Comment:
Where is this coming from? I wasn't able to trace it

Reply:
Comes from the `cookieless_testable` concern